osx/KBKit/KBKit/Component/KBHelperTool.m

```diff
@@ -74,6 +74,27 @@ - (void)log:(MPLogLevel)level format:(NSString *)format, ... {
   va_end(args);
 }
 
+- (void)doInstallAlert:(KBSemVersion *)bundleVersion runningVersion:(KBSemVersion *)runningVersion {
+  if ([runningVersion.version length] == 0) {
+    // No need to show anything if the user is explicitly choosing to install
+    // and just clicked on something.
+    return;
+  }
+
+  NSString *alertText = @"Keybase is about to upgrade the Keybase file system, allowing end-to-end encrypted files from right inside your Finder.";
+  NSString *infoText = @"";
+  if ([bundleVersion isOrderedSame:[KBSemVersion version:@"1.0.30"]]) {
+    alertText = @"New Keybase feature: multiple users in macOS";
+    infoText = @"The Keybase file system (KBFS) has been mounted at /keybase.  This doesn't work for multiple users, so we're moving things around.  You'll now have a unique mountpoint in your home directory.  Follow the new \"Favorite\" in Finder to get to it. The first local user to run Keybase will also have a link to it from the old /keybase location.\n\nPlease provide your password at the secure prompt when requested to unlock this new, open-source-powered feature!";
```

Comment:
> (KBFS) has been mounted at /keybase

I'm hesitant to use words like "mounted".

Also, I was confused when I first read this, interpreting "has been mounted" as "we've successfully mounted". Maybe "(KBFS) was previously available at /keybase."

> You'll now have a unique mountpoint in your home directory.

"Your KBFS will now be available in your home directory at ~/keybase."

> The first local user to run Keybase will also have a link to it from the old /keybase

Not sure we can expect users to know what a "link" is in this context. I'd say "For convenience, the first user to run Keybase on this computer will also be able to access KBFS from the old /keybase location."

Reply:
This is mostly @malgorithms text, I don't want to change anything without his approval.  @malgorithms?

Reply:
@malgorithms: thanks, I'm fine with the rewrite but I think we really need something about the old paths still working, I don't want to risk freaking anyone out who has custom scripts, etc.  How about adding a sentence like:

> And if you're the only Keybase user on this computer, the old directory will continue to work as well.

Reply:
sounds good to me!

Reply:
I like the rewrite, and I agree with strib's amendment.